docs/tutorials/multi-d-datasets.ipynb

```diff
@@ -202,7 +202,9 @@
    "metadata": {},
    "source": [
     "Note the important call to `copy()`.\n",
-    "If we omit it, `d` will just be a multi-dimensional slice of the larger volume (which is kept alive), wasting memory and preventing further modification, such as insertion of other variables."
+    "If we omit it, `d` will just be a multi-dimensional slice of the larger volume (which is kept alive), wasting memory and preventing further modification, such as insertion of other variables.\n",
+    "\n",
+    "Note also that if we had also sliced `Dim.Y` the result would not contain the data for `'bob'` since this item does not depend on Y."
    ]
   },
   {
```

Comment:
Could not comment on the right line, since you did not change it, but since we do not have `sort` anymore, we should remove it from
"Operations like `concatenate` and `sort` work just like with one-dimensional datasets."
below.